tasks/clean-references.mjs

```diff
@@ -0,0 +1,134 @@
+import references from '../src/references/raw-references.json' assert { type: 'json' };
+import { writeFileSync } from 'fs';
+import {
+  API_CATEGORIES,
+  API_SUB_CATEGORIES
+} from '../src/data/api-categories.mjs';
+
+const cleanReferences = {};
+const categoryNodes = [];
+
+const categories = Object.values(API_CATEGORIES).concat(
+  Object.values(API_SUB_CATEGORIES)
+);
```

Comment:
not blocking: I think if we move these consts closer to where it was being used, it would be easier for others to understand how they are being used maybe?